### PR TITLE
update scrollto method for ckeditor fields

### PR DESCRIPTION
**Pull request by vsteenhoudt**

### CODE REVIEW

File: src/components/Modal.js [hunk 1]
```diff
@@ -2,10 +2,11 @@ import React, { useState, useEffect, useRef } from "react";
 import { Modal as M } from "react-responsive-modal";
 import { Trans } from "@lingui/macro";
 import CloseIcon from "./icons/close";
+import { constSelector } from "recoil";
```
Comment (thomasmattheussen) [inline src/components/Modal.js]: what's this?

File: src/iframe-work-arounds.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ const proto = {
             onReady: () => this.onIFrameResizerReady()
         };
 
+        window.scrollToOrig = window.scrollTo;
```
Comment (thomasmattheussen) [inline src/iframe-work-arounds.js:7]: does this still need to be here?

Comment (vsteenhoudt) [reply src/iframe-work-arounds.js:7]: Yes, otherwise it will fail outside of the iframe

File: src/components/Modal.js (hunk 1 above)
Comment (vsteenhoudt) [reply src/components/Modal.js]: I dunno :)

**Pull request merged**